client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -8,7 +8,7 @@ import { useEffect } from 'react';
 import Navbar from '../../components/Navbar/Navbar';
 import { Box, Typography, Button } from '@material-ui/core';
 import AvatarDisplay from '../../components/AvatarDisplay/AvatarDisplay';
-import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
+// import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
```

Comment:
If we are not going to be using this component, I guess we can delete the comment :)